PR: https://github.com/odoo/odoo/pull/

From: a81942cf7efc6ec75b0fb1d30e243fc7a5b6f529
From: William Braeckman
Date: 2021-07-09 12:51:26

Breaking data model changes scores: del:4 + add:0, change matches:
-    _inherit = 'fleet.vehicle.odometer'
-    _inherit = 'fleet.vehicle.log.contract'
-    purchaser_employee_id = fields.Many2one(
-    fleet_is_internal = fields.Boolean(related='vehicle_id.fleet_is_internal')

Total Changes: 97

[MOV] hr_fleet: Split model files according to coding guidelines

Applies the same changes as 6747402a96d3bd6013aa9e0ec29bca17c8ffc4a9 to
hr_fleet

Task ID: 2415309

================================= pseudo patch: =================================

--- a/addons/hr_fleet/models/__init__.py
+++ b/addons/hr_fleet/models/__init__.py
@@ -6,4 +6,6 @@ from . import res_users
 from . import fleet_vehicle_assignation_log
 from . import fleet_category
 from . import fleet_vehicle
-from . import fleet_vehicle_cost
+from . import fleet_vehicle_log_contract
+from . import fleet_vehicle_log_services
+from . import fleet_vehicle_odometer

--- a/addons/hr_fleet/models/fleet_vehicle.py
+++ b/addons/hr_fleet/models/fleet_vehicle.py
@@ -165,11 +165,3 @@ class FleetVehicle(models.Model):
                         partners_to_unsubscribe.append(employee.user_id.partner_id.id)
                     vehicle.message_unsubscribe(partner_ids=partners_to_unsubscribe)
         return super().write(vals)
-
-class FleetVehicleOdometer(models.Model):
-    _inherit = 'fleet.vehicle.odometer'
-
-    driver_employee_id = fields.Many2one(
-        related='vehicle_id.driver_employee_id', string='Current Driver (Employee)',
-        readonly=True,
-    )

--- a/None
+++ b/addons/hr_fleet/models/fleet_vehicle_log_contract.py
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+class FleetVehicleLogContract(models.Model):
+    _inherit = 'fleet.vehicle.log.contract'
+
+    purchaser_employee_id = fields.Many2one(
+        related='vehicle_id.driver_employee_id',
+        string='Current Driver (Employee)',
+    )
+    fleet_is_internal = fields.Boolean(related='vehicle_id.fleet_is_internal')

--- a/addons/hr_fleet/models/fleet_vehicle_cost.py
+++ b/addons/hr_fleet/models/fleet_vehicle_log_services.py
@@ -3,15 +3,6 @@
 
 from odoo import api, fields, models
 
-class FleetVehicleLogContract(models.Model):
-    _inherit = 'fleet.vehicle.log.contract'
-
-    purchaser_employee_id = fields.Many2one(
-        related='vehicle_id.driver_employee_id',
-        string='Current Driver (Employee)',
-    )
-    fleet_is_internal = fields.Boolean(related='vehicle_id.fleet_is_internal')
-
 class FleetVehicleLogServices(models.Model):
     _inherit = 'fleet.vehicle.log.services'
 

--- a/None
+++ b/addons/hr_fleet/models/fleet_vehicle_odometer.py
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class FleetVehicleOdometer(models.Model):
+    _inherit = 'fleet.vehicle.odometer'
+
+    driver_employee_id = fields.Many2one(
+        related='vehicle_id.driver_employee_id', string='Current Driver (Employee)',
+        readonly=True,
+    )
